autometa/taxonomy/lca.py

```diff
@@ -74,10 +74,13 @@ class LCA(NCBI):
 
     """
 
-    def __init__(self, dbdir: str, verbose: bool = False, cache: str = ""):
+    def __init__(
+        self, dbdir: str, verbose: bool = False, cache: str = "", dbType: str = "gtdb"
+    ):
         super().__init__(dbdir, verbose=verbose)
         self.verbose = verbose
         self.dbdir = dbdir
+        self.dbType = dbType
```

Comment:
CamelCase should not be used as snake_case is convention and is used throughout. For consistency, this should either be `db_type` or `dbtype`